### PR TITLE
Add option to append blank frames when recording video file

### PR DESCRIPTION
**Pull request by Haichao-Zhang**
For some environments, it is difficult to tell the episode boundary in the recorded video.
Add option to append blank frames at episode end when recording video files to make it easier.

### CODE REVIEW

File: alf/utils/video_recorder.py [hunk 1]
```diff
@@ -299,6 +306,15 @@ def capture_frame(self,
                 else:
                     self._encode_image_frame(frame)
 
+            if self._append_blank_frames > 0 and is_last_step:
+                if self._blank_frame is None:
+                    self._blank_frame = np.zeros_like(self.last_frame)
```
Comment (hnyu) [inline alf/utils/video_recorder.py]: self._last_frame

File: alf/utils/video_recorder.py [hunk 2]
```diff
@@ -299,6 +306,15 @@ def capture_frame(self,
...
+                for _ in range(self._append_blank_frames):
+                    if self.ansi_mode:
```
Comment (hnyu) [inline alf/utils/video_recorder.py]: This branch appears multiple times in this file. Can use a function to wrap it.

**Review by hnyu — CHANGES_REQUESTED** [review]

File: alf/utils/video_recorder.py (hunk 2 above)
Comment (Haichao-Zhang) [reply alf/utils/video_recorder.py]: done

File: alf/utils/video_recorder.py (hunk 1 above)
Comment (Haichao-Zhang) [reply alf/utils/video_recorder.py]: changed

**Review by hnyu — APPROVED** [review]

**Pull request merged**